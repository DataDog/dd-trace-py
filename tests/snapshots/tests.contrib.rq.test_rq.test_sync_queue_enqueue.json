[[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.job_add1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "rq",
      "job.func_name": "tests.contrib.rq.jobs.job_add1",
      "job.id": "e946f2ef-0642-4fd7-a146-f14539ff4350",
      "language": "python",
      "queue.name": "sync-q",
      "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
      "span.kind": "producer"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 55795
    },
    "duration": 1792343,
    "start": 1691167585360156118
  },
     {
       "name": "rq.job.perform",
       "service": "rq",
       "resource": "tests.contrib.rq.jobs.job_add1",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "_dd.p.dm": "-0",
         "component": "rq",
         "job.id": "e946f2ef-0642-4fd7-a146-f14539ff4350"
       },
       "duration": 278700,
       "start": 1691167585361209860
     }]]
