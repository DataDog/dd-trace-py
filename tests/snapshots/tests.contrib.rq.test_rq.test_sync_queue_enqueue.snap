[[{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.job_add1"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1631049654148153700
   "duration" 23705500
   "meta" {"runtime-id" "6e75d0807aa04f78aea66a630db7ac7f"
           "queue.name" "sync-q"
           "job.id" "5f0176f9-a10e-43a3-bc03-aced1bfae677"
           "job.func_name" "tests.contrib.rq.jobs.job_add1"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 684
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.job.perform"
       "service" "rq"
       "resource" "tests.contrib.rq.jobs.job_add1"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1631049654165756500
       "duration" 1037100
       "meta" {"job.id" "5f0176f9-a10e-43a3-bc03-aced1bfae677"}}]]
